File: app/src/main/java/bj/discogsbrowser/model/labelrelease/RootLabelResponse.java

package bj.discogsbrowser.model.labelrelease;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import bj.discogsbrowser.model.common.Pagination;

public class RootLabelResponse
{

    @SerializedName("pagination")
    @Expose
    private Pagination pagination;
    @SerializedName("releases")
    @Expose
    private List<LabelRelease> labelReleases = null;

    public Pagination getPagination()
    {
        return pagination;
    }

    public void setPagination(Pagination pagination)
    {
        this.pagination = pagination;
    }

    public List<LabelRelease> getLabelReleases()
    {
        return labelReleases;
    }

    public void setLabelReleases(List<LabelRelease> labelReleases)
    {
        this.labelReleases = labelReleases;
    }

}

File: app/src/main/java/bj/discogsbrowser/model/labelrelease/RootLabelResponse.kt
package bj.discogsbrowser.model.labelrelease

import bj.discogsbrowser.model.common.Pagination
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class RootLabelResponse(val pagination: Pagination,
                             @SerializedName("releases") val labelReleases: List<LabelRelease>)